markdown/markdown.php

```diff
@@ -238,6 +237,16 @@ function Markdown_Parser() {
 		asort($this->span_gamut);
 	}
 
+	function Markdown_Parser() {
```

Comment:
I don't think this is necessary; subclasses should call `parent::__construct()`

Reply:
I think that it is the best solution to keep your code and assure the future compatibility with PHP7. Also the solution makes low footprint in your code.

If you prefer you can rewrite the code. If it is possible, always I respect the authors code.